src/Raven.Server/Utils/TypeConverter.cs

```diff
@@ -226,7 +226,7 @@ private static object ToBlittableSupportedType(object root, object value, bool f
 
                 return @object;
             }
-
+            // looks like best option
             if (value is IEnumerable<char> charEnumerable)
                 return new string(charEnumerable.ToArray());
```

Comment:
That will almost always be either `string` of `char[]`, we can check for them explicitly

Reply:
done, i am checking for
if (value is char[] chars) AND if (value is string)
instead